docs/apache-airflow/authoring-and-scheduling/dynamic-task-mapping.rst

```diff
@@ -177,9 +177,8 @@ Mapping with non-TaskFlow operators
 It is possible to use ``partial`` and ``expand`` with classic style operators as well. Some arguments are not mappable and must be passed to ``partial()``, such as ``task_id``, ``queue``, ``pool``, and most other arguments to ``BaseOperator``.
```

Comment:
Maybe explicitly mention something about the example `sum_it` being a "reduce" task?